plugin_template.rb

```diff
@@ -1,6 +1,18 @@
-# Start with...
-# rails plugin new path/to/my_plugin --full -m path/to/plugin_template.rb --skip-spring --skip-active-record --skip-action-cable
-#
+=begin
+To generate a new plugin, run:
+
+rails plugin new path/to/my_plugin \
+  --template=https://raw.githubusercontent.com/workarea-commerce/workarea/v3.4-stable/plugin_template.rb \
+  --full \
+  --skip-sprint \
```

Comment:
```suggestion
  --skip-spring \
```

Reply:
Can't blame a guy for tryin'. #agilehulk